### DISCUSSION TITLE
How to become middleware for .Net compilation

### DISCUSSION THREAD

**Discussion by Yeah69** [adamralph/minver#514] [Q&A] [answered]
Hi,

first of all, thank you I like the project a lot and it helped me greatly already.
Obviously MinVer works like a middleware during compilation. Do you have some pointers for me on how to do that?
I am working on a project where could use that.

Regards
Dima

**Comment by adamralph** [comment, marked as answer]
Hi @Yeah69, it's not part of _compilation_ exactly, but it is part of the build process. The MinVer package ships with an MSBuild target which injects itself into the build process here — https://github.com/adamralph/minver/blob/main/MinVer/build/MinVer.targets#L14
